src_py/pkgdata.py

```diff
@@ -22,7 +22,24 @@ def getResource(identifier, pkgname=__name__):
 import os
 
 try:
-    from pkg_resources import resource_stream, resource_exists
+    if sys.version_info[:2] > (3, 8):
+        from importlib.resources import files
+
+        def resource_exists(_package_or_requirement, _resource_name):
+            return files(_package_or_requirement).joinpath(_resource_name).is_file()
+
+        def resource_stream(_package_or_requirement, _resource_name):
+            ref = files(_package_or_requirement).joinpath(_resource_name)
+            return ref.open('rb')
+    else:
+        from importlib import resources
+
+        def resource_exists(_package_or_requirement, _resource_name):
+            return resources.is_resource(_package_or_requirement, _resource_name)
+
+        def resource_stream(_package_of_requirement, _resource_name):
+            return resources.read_binary(_package_of_requirement, _resource_name)
```

Comment:
```suggestion
            return resources.open_binary(_package_of_requirement, _resource_name)
```
`read_binary` returns a `bytes` instance, but we need `open_binary` which returns a filelike object